tests/e2e/test_post_and_cancel_order_live_e2e.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import os
+from dataclasses import asdict
+
+import pytest
+from eth_account import Account
+from eth_typing import Address
+
+from cow_py.codegen.abi_handler import dict_keys_to_camel_case
+from cow_py.common.chains import Chain
+from cow_py.common.config import SupportedChainId
+from cow_py.common.constants import CowContractAddress
+from cow_py.contracts.domain import domain
+from cow_py.contracts.order import Order
+from cow_py.contracts.sign import SigningScheme, sign_order, sign_order_cancellation
+from cow_py.order_book.api import OrderBookApi
+from cow_py.order_book.config import OrderBookAPIConfigFactory
+from cow_py.order_book.generated.model import (
+    EcdsaSignature,
+    EcdsaSigningScheme,
+    OrderCancellations,
+    OrderCreation,
+    OrderQuoteRequest,
+    OrderQuoteSide1,
+    OrderQuoteSideKindSell,
+    SigningScheme as ModelSigningScheme,
+    TokenAmount,
+)
+
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+E2E_SEPOLIA_TESTING_EOA_PRIVATE_KEY = os.getenv(
+    "E2E_SEPOLIA_TESTING_EOA_PRIVATE_KEY", ""
+)
+
+COW_TOKEN_SEPOLIA_ADDRESS = "0x0625aFB445C3B6B7B929342a04A22599fd5dBB59"
+WETH_SEPOLIA_ADDRESS = "0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14"
+
+E2E_SEPOLIA_TESTING_EOA = Account.from_key(E2E_SEPOLIA_TESTING_EOA_PRIVATE_KEY)
+E2E_SEPOLIA_TESTING_EOA_ADDRESS: Address = E2E_SEPOLIA_TESTING_EOA.address
+
+
+@pytest.mark.vcr
```

Comment:
Why is vcr being used here (and also together with the other yml file)?

Reply:
mentioned that [here ](https://github.com/cowdao-grants/cow-py/pull/12#issuecomment-2340448804), ultimately we're choosing to go with testing on mainnet though